src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -1548,16 +1547,10 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
 
           let doc = p.TextDocument
           let filePath = doc.GetFilePath() |> Utils.normalizePath
-          let file = volaTileFile filePath doc.Text (Some doc.Version) DateTime.UtcNow
+          // We want to try to use the file system's datetime if available
+          let file = VolatileFile.Create(filePath, doc.Text, (Some doc.Version))
           updateOpenFiles file
-
-          Async.Start(
-            async {
-              do! Async.Sleep 100
-              forceGetTypeCheckResults filePath |> ignore
-            }
-          )
-
+          forceGetTypeCheckResults filePath |> ignore
```

Comment:
Instead of just waiting around, lets start type checking since we'll cancel it on the next request anyway